website/docs/getting_started/flashing_firmware/flashing_firmware_cli.md

```diff
@@ -0,0 +1,77 @@
+---
+id: flashing_firmware_cli
+title: Flashing Firmware – CLI
+sidebar_label: Command Line Interface
+slug: /getting_started/flashing_firmware/cli
+---
+
+:::caution
+Be very careful to install the correct load for your board. In particular the popular 'T-BEAM' radio from TTGO is not called 'TTGO-Lora' (that is a different board). So don't install the 'TTGO-Lora' build on a TBEAM, it won't work correctly.
+:::
+
+## Linux
+
+Requires `Python` and `pip`
+
+1. Download and unzip the latest Meshtastic firmware [release](https://github.com/meshtastic/Meshtastic-esp32/releases).
+2. `pip install --upgrade esptool` - Installs esptool on your machine
+3. Connect your radio to your USB port
+4. `esptool.py chip_id` - Confirm that your device is talking to your PC by running
+You should see something like:
+```
+mydir$ esptool.py chip_id
+esptool.py v2.6
+Found 2 serial ports
+Serial port /dev/ttyUSB0
+Connecting....
+Detecting chip type... ESP32
+Chip is ESP32D0WDQ6 (revision 1)
+Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
+MAC: 24:6f:28:b5:36:71
+Uploading stub...
+Running stub...
+Stub running...
+Warning: ESP32 has no Chip ID. Reading MAC instead.
+MAC: 24:6f:28:b5:36:71
+Hard resetting via RTS pin...
+```
+5. `cd` into the directory where you unzipped the latest release.
+6. Install the correct firmware for your board with `device-install.sh -f firmware-BOARD-VERSION.bin`
+```
+./device-install.sh -f firmware-heltec-1.2.x.bin
+```
+7. To update a the firmware on an existing Meshtastic device, run `device-update.sh -f firmware-BOARD-VERSION.bin`
+```
+./device-update.sh -f firmware-heltec-1.2.x.bin
+```
+
+
+
+## MacOS
+
+Installing on OS X through the commandline uses the same method as linux, but requires a slightly different method to install Python, pip and esptool.
+
+OS X (10.15.3/Catalina) comes with Python 2.7 installed, but not pip.
+
+:::note
+If you don't know if Homebrew is installed on your Mac, open terminal and enter the command `brew -v` if it doesn't work [install Homebrew](https://brew.sh) before continuing.
+:::
+
+1. `brew install pyenv` - Installs PyEnv
+2. `pyenv install 3.7.7` - To install and select Python 3.7.7
+3. `pyenv global 3.7.7` - Selects the new version of Python
+4. `brew install pip` - Installs pip3
+5. `pip3 install --upgrade esptool` - Installs esptool
```

Comment:
Same deal, also don't have notes inline with commands

Reply:
Reworking many commands to do that. I think we may have found our first order of style guide business! using code blocks for commands, not just ticks.

Reply:
Yeah, once we get past this first major hurdle of bringing in and refining all of the required docs, I'll write a style guide (please nag me if I forget)